pkg/cableengine/syncer/syncer.go

```diff
@@ -209,6 +210,9 @@ func (i *GatewaySyncer) generateGatewayObject() *v1.Gateway {
 				latencyInfo := i.healthCheck.GetLatencyInfo(&connection.Endpoint)
 				if latencyInfo != nil {
 					connection.Latency = latencyInfo.Spec
+					cable.RecordConnectionLatency(localEndpoint.Spec.CableName, &localEndpoint.Spec, &connection.Endpoint,
+						float64(latencyInfo.Spec.LastRTT/uint64(time.Second)))
```

Comment:
Shouldn't we only record this if connected (ie `connection.Status == v1. Connected`)?